core/src/main/java/com/linecorp/armeria/common/util/Sampler.java

```diff
@@ -80,7 +53,45 @@ public boolean isSampled() {
      *
```

Comment:
Can we remove the paragraph above, because we do not provide `BoundarySampler`?

Reply:
Thank you for review. fiexed in 6459312.

Reply:
Could you also remove the paragraph: `The sample returned is good for ....`.